quark/subproject.py

```diff
@@ -902,7 +902,7 @@ def _resolve_job_id(self, project: str, ref: str, job_name: str) -> str:
 
         # Jobs in latest pipeline
         req = urllib.request.Request(
-            url="%s/api/v4/projects/%s/pipelines/%s/jobs" % (
+            url="%s/api/v4/projects/%s/pipelines/%s/jobs?scope=success" % (
```

Comment:
This may break some use case, because one may be downloading the artifacts of a failed job too. Do we need to support that and do the pagination loop in case of failure in the first page?

Reply:
Also, `per_page=50` is arbitrary but may do the trick making the pagination effect more limited.